actix-http/src/body/none.rs

```diff
@@ -40,14 +40,4 @@ impl MessageBody for None {
     ) -> Poll<Option<Result<Bytes, Self::Error>>> {
         Poll::Ready(Option::None)
     }
-
-    #[inline]
-    fn is_complete_body(&self) -> bool {
-        true
-    }
-
-    #[inline]
-    fn take_complete_body(&mut self) -> Bytes {
-        Bytes::new()
-    }
```

Comment:
I think I understand why you removed this; the use case for these methods also should go along with checking the size hint for `None` and bailing and, as such, these methods will/should never be called. So there are two options:
- Keep them and allow for some unexpected, exotic use case where they _are_ reasonably called.
- Use a custom panic message that indicates that size hint should be checked first.